.circleci/config.yml

```diff
@@ -41,12 +41,10 @@ jobs:
           mkdir -p /go/out
           mkdir /home/circleci/logs
       - checkout
+      - run: make submodule-sync
       - attach_workspace:
           at:  /go
       - run: bin/testEnvRootMinikube.sh start
-      - restore_cache:
```

Comment:
We still need the cache for envoy and other downloads.

Reply:
I thought you mentioned that's dealt with through the attach_workspace